policyengine_us/parameters/gov/states/co/tax/income/subtractions/collegeinvest_contribution/max_amount.yaml

```diff
@@ -9,19 +9,33 @@ metadata:
     # Subtraction amount is only specified in the Legal Code but not in the Tax Forms
     - title: C.R.S. 39-22-104(4)(j)(II)(B)
       href: https://law.justia.com/codes/colorado/2022/title-39/article-22/part-1/section-39-22-104/
```

Comment:
Add IRS uprating rounded to $100 per C.R.S. 39-22-104(4)(j)(II)(C), with a comment that this approximates the true uprating which is for college tuition, room, and board